services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -38,6 +52,7 @@ export const ReportPageWrapper = () => {
   const handleBlur = (data: any) => {
     if (!report) return;
     setAnswers(data);
+    setLastSavedTime(getLocalHourMinuteTime());
```

Comment:
I think this needs to be slightly adjusted. If setAnswers fails, it will need to update the banner with a failure, thus it needs to have a way of handling success here.

Likely setAnswers should update the store with a lastSavedTime if successful